domains/certificates/Query.API/API/ContractService/ContractServiceImpl.cs

```diff
@@ -161,7 +164,6 @@ private static MeteringPointType Map(MeterType type)
         var contractsByGsrn =
             await unitOfWork.CertificateIssuingContractRepo.GetByGsrn(issuingContracts.Select(c => c.GSRN).ToList(),
                 cancellationToken);
-
         foreach (var updatedContract in contracts.Contracts)
```

Comment:
Does this have the same issue as mentioned previously?

Reply:
fixed in 1eb3207b